core/integration_test/hws_integration_test.jl

```diff
@@ -0,0 +1,36 @@
+@testitem "HWS model integration test" begin
+    using SciMLBase: successful_retcode
+    using Arrow
+
+    toml_path = normpath(@__DIR__, "../../../hws_2024_7_0/hws.toml")
```

Comment:
```suggestion
    toml_path = normpath(@__DIR__, "../../models/hws_2024_7_0/hws.toml")
```

This folder is already in .gitignore, this is where I keep all models. Perhaps good to use that.